extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -131,118 +131,118 @@ public function adContextDataProvider() {
 			],
 			[
 				'titleMockType' => 'article',
-				'flags' => ['wgWikiDirectedAtChildrenByFounder'],
-				'expectedOpts' => [],
+				'flags' => [ 'wgWikiDirectedAtChildrenByFounder' ],
+				'expectedOpts' => [ ],
 				'expectedTargeting' => [
 					'newWikiCategories' => [ 'test' ],
 					'esrbRating' => 'ec'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "esrbRating" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)